staging/src/k8s.io/component-base/cli/flag/flags.go

```diff
@@ -47,7 +47,6 @@ func WarnWordSepNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedNam
 func InitFlags() {
 	pflag.CommandLine.SetNormalizeFunc(WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
-	pflag.Parse()
```

Comment:
I worry about removing this line. Not at my computer but think it could effect others using this function. Will check on this in the morning.

Reply:
I have tested that command line arguments will not be resolved properly if  `pflag.Parse()`  is not removed. This may be related to `cobra` ?

Reply:
If we look at https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/apiserver.go#L36-L39, and similar references in cmd/cloud-controller-manager/cloud-controller-manager.go, cmd/kube-controller-manager/controller-manager.go, cmd/kube-proxy/proxy.go (this PR), cmd/kube-scheduler/scheduler.go, cmd/kubectl/kubectl.go and cmd/kubemark/hollow-node.go (basically all the previous references to InitFlags), they alll stopped referring to it because of it's pflag.Parse(). There are no references to InitFlag in the code base today. So looks like removing pflag.Parse() seems to be the right thing to do and would then enable others to start using it again.